pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -25,8 +27,40 @@ func Status(options *Options, settings *env.Settings) error {
 	return status(kc, options, settings.Namespace)
 }
 
+func statusFormatted(kc *kudo.Client, options *Options, ns string) error {
+	instance, err := kc.GetInstance(options.Instance, ns)
+	if err != nil {
+		return err
+	}
+	if instance == nil {
+		return fmt.Errorf("instance %s/%s does not exist", ns, options.Instance)
+	}
+
+	var o []byte
+	if strings.ToLower(options.Output) == "yaml" {
```

Comment:
nit: maybe we should check that the `output` parameter has the right value and put both `yaml` and `json` into constants since we're probably going to reuse these for other commands

Reply:
Yeah... I ended up refactoring this quite a bit and put the whole yaml/json marshalling into a new package...